src/popup/helper.js

```diff
@@ -113,3 +113,39 @@ export const licenseInfo = {
     licenseDescription: 'This work has been marked as dedicated to the public domain.',
   },
 };
+
+/* Object that holds the data to be displayed in the tooltip for each license. */
+export const tooltipInfo = {
+  by: {
+    tooltipHeading: `License CC BY`,
+    tooltipContent: `<p><i class='icon ${licenseInfo.by.licenseIcon}'></i> ${licenseInfo.by.licenseDescription}</p>`,
+  },
+  bync: {
+    tooltipHeading: `License CC BY-NC`,
+    tooltipContent: `<p><i class='icon ${licenseInfo.by.licenseIcon}'></i> ${licenseInfo.by.licenseDescription}<br/><i class='icon ${licenseInfo.nc.licenseIcon}'></i> ${licenseInfo.nc.licenseDescription}</p>`,
+  },
+  bysa: {
+    tooltipHeading: `License CC BY-SA`,
+    tooltipContent: `<p><i class='icon ${licenseInfo.by.licenseIcon}'></i> ${licenseInfo.by.licenseDescription}<br/><i class='icon ${licenseInfo.sa.licenseIcon}'></i> ${licenseInfo.sa.licenseDescription}</p>`,
+  },
+  bynd: {
+    tooltipHeading: `License CC BY-ND`,
+    tooltipContent: `<p><i class='icon ${licenseInfo.by.licenseIcon}'></i> ${licenseInfo.by.licenseDescription}<br/><i class='icon ${licenseInfo.nd.licenseIcon}'></i> ${licenseInfo.nd.licenseDescription}</p>`,
```

Comment:
Instead of duplicating the information here, we can use the `licenseInfo` object. The task of creating icon and paragraph tags can be delegated to `JavaScript`.

Reply:
Are you suggesting to remove the entire `tooltipInfo` object itself? We would need that for the headings right? Since they're not present in the `licenseInfo` object.

Reply:
Heading can be easily created using the license name.

``` javascript
    // license heading
    const heading = document.createElement('h5');
    heading.classList.add('b-header');
    heading.innerText =
      licenseName === 'cc0' || licenseName === 'pdm'
        ? licenseName.toUpperCase()
        : `License CC ${licenseName.toUpperCase()}`;
```